Library/Homebrew/cask/cask.rb

```diff
@@ -361,7 +362,6 @@ def to_h
     # @private
     def to_internal_api_hash
       api_hash = {
-        "token"              => token,
```

Comment:
This doesn't need to be included in the hash because it's already the key of each object. Not sure how I missed this before.